### PR TITLE
Devdocs 2863

### PR DESCRIPTION
**Pull request by markcmurphy**
# [DEVDOCS-2863](https://jira.bigcommerce.com/browse/DEVDOCS-2863)

## What changed?
* add a note regarding depreciation of `google_shopping` and `marketing` pair
* add a note about newly added `google` platform for type `marketing` and `marketplace`

### CODE REVIEW

File: docs/api-docs/channels/channels-extending-existing.md [hunk 1]
```diff
@@ -163,8 +163,22 @@ Accept: application/json
...
+| `google` | `marketplace` |
+| `custom` | `storefront`, `pos`, `marketing`, `marketplace` |
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+> ### Note
+>
+> \* The `google_shopping` platform is depreciated and should **not** be used. Support for the types `marketing` and `marketplace` on the `google` platform are being added.
```
Comment (traci707) [inline docs/api-docs/channels/channels-extending-existing.md:177]: We discourage using  `google_shopping`  as BigCommerce has deprecated this platform. Instead, we have added support for `marketing` and `marketplace` types on the `google` platform.

Comment (traci707) [reply docs/api-docs/channels/channels-extending-existing.md:177]: I wanted to replace the passive voice.

**Review by traci707 — COMMENTED** [review]
Gave a suggestion for rewriting the note. Feel free to modify it.

File: docs/api-docs/channels/channels-extending-existing.md (hunk 1 above)
Comment (markcmurphy) [reply docs/api-docs/channels/channels-extending-existing.md]: Thanks, I made those changes.

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**